packages/rate-limit/README.md

```diff
@@ -0,0 +1,32 @@
+# @tinyhttp/pug
+
+[![npm (scoped)][npm-badge]](https://npmjs.com/package/@tinyhttp/pug) [![npm][dl-badge]](https://npmjs.com/package/@tinyhttp/pug) [![][web-badge]](https://tinyhttp.v1rtl.site/mw/pug)
```

Comment:
ik it's a draft pr but you forgot to rename `pug` to `rate-limit` (easily done via find/replace)